recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # Here we can see that the 'PLANET_SHAPIRO' entry if 'Y' so we set it to True.
</pre><p>But in the file, it is 'N'!!!!</p><p>Something is not consistent</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>